pymc/sampling_jax.py

```diff
@@ -213,9 +216,7 @@ def sample_numpyro_nuts(
         dims = {}
 
     if random_seed is None:
-        random_seed = model.rng_seeder.randint(
-            2**30, dtype=np.int64, size=chains if chains > 1 else None
-        )
+        random_seed = model.rng_seeder.randint(2**30, dtype=np.int64)
```

Comment:
@twiecki The fix is here. Only one seed is created instead of one per `chain`

Reply:
No, they are split below and also in the jitter method

Reply:
:+1: